R/logging.R

```diff
@@ -63,6 +63,157 @@ message_wrap <-
     invisible(msg)
   }
 
+# issue cataloger --------------------------------------------------------------
+tune_env <-
+  rlang::new_environment(
+    data = list(progress_env = NULL, progress_active = FALSE, progress_catalog = NULL)
+  )
+
+# determines whether a supplied bar id references a tuning process that
+# uses the issue cataloger.
+uses_catalog <- function() {
+  isTRUE(tune_env$progress_active && !is_testing())
+}
+
+# determines whether a supplied bar id references an issue cataloger that
+# has not yet been terminated. this function is intended to guard calls to
```

Comment:
I think you need to update these comments? i.e. they reference a "supplied bar id" but there aren't arguments to this anymore

Reply:
Ah, thank you!